Setup.pm

```diff
@@ -65,17 +62,22 @@ sub make_path {
 sub create_internet_shortcut {
     my $target  = shift;
     my $lnkPath = shift;
+    my $iconPath = shift;
 
     if ( -e $lnkPath ) {
         unlink $lnkPath;
     }
 
-    #print "Creating internet shortcut: $lnkPath -> $target\n";
-    my $link = Win32::Unicode::InternetShortcut->new;
-    $link->save($lnkPath, $target);
-    $link->load($lnkPath);
+    my $str = <<END;
+[InternetShortcut]
+URL=${target}
+IconFile=${iconPath}
+IconIndex=0
+END
 
-    ($link->{url} eq $target) || die "Not the same url\n";
+    open(FH, '>', $lnkPath) or die "Couldn't create internet shortcut '$target': $!";
+    print FH $str;
+    close(FH);
```

Comment:
I had thought about doing this, but didn't want to risk any unknowns. I'm happy to go with it; Please have Pete remove the Win32::Unicode::InternetShortcut related dep(s) from the project. Also, please be sure that this works correctly in a test environment and that the same icon should be used for both types of shortcuts. Personally, I would use the website ico file for the internet shortcut.

Reply:
I tested it quite thoroughly.  It ate up most of the time of doing this actually. :)  The icon can be re-used between both shortcuts.  But as you suggested I added our web icon.  It is a little strange to have them residing in this repo IMO.  But I created a follow-up story to integrate at least our state tool icon in our .exe file.  Once that is done, we can simply link to the state tool exe.

And I'll ping Pete.

Reply:
Follow-up story is here: https://www.pivotaltracker.com/story/show/175964852

Reply:
Consider if it's worth downloading the ico (to avoid the need for it to be stored/updated in this repo): https://platform.activestate.com/img/favicon.ico